### DISCUSSION TITLE
Cannot add new connection

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#454] [General]
First.. I would like to thank you for your efforts on this project. I am a big fan of Machinaris -- so thank you!

I juts download the latest version so i can farm Maize and after I got everything setup.. I tried to add a new connection for maize to start to sync the blockchain... but its like the "Add New Connection" button is broken - it juts does not seem to do anything at all.

not a huge deal... but more of an FYI

**Comment by LeroyINC** [comment]
also... if I type in the <IPaddress>:<port number> in the add new connection box.. then juts hit the enter key.. it looks like it processing the adding new connection... then I get a message that says "Connection removed from maize!"

**Comment by guydavis** [comment]
> 

Hi, thanks for the report.  Well, the 'Add Connection' is not immediate so perhaps on refresh in a few minutes you will see the connection appear.  However, it could be that you have network issues.

Just tested this again with this example Maize node: 109.155.166.187:8644

![20211130_102330](https://user-images.githubusercontent.com/12936/144097042-e38015b7-0f81-42ef-a92e-9bdc6b574116.png)

which gave: 

![image](https://user-images.githubusercontent.com/12936/144097184-febc7865-0d38-4ed4-b52c-8c45517b392c.png)

Regarding you hitting the Enter key, probably the default form submit button is 'Disconnect Selected' so you will need to click the 'Add Connection' button.  Thanks for the report though, I'll keep this in mind as a possible future UI improvement around keyboard focus.

**Comment by LeroyINC** [comment]
Thanks for the response...

it looks like the network ip/port I was using was not valid... when I tried to add the same one you did it worked.

**Reply by guydavis** [reply]
Great, FYI here are two excellent sites to find nodes for each blockchain:

- https://alltheblocks.net/
- https://xchforks.com/
